Vue project LSP not work

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

/Users/d1y/Library/Application Support/Zed/extensions/work/vue

<img width="1187" alt="image" src="https://github.com/zed-industries/zed/assets/45585937/0172c73d-d9dc-449d-b429-8b7b739b9943">


<img width="477" alt="image" src="https://github.com/zed-industries/zed/assets/45585937/835fcbbd-32f9-4986-a6c0-a5bc2ce3a99b">


```log
Caused by:
    No such file or directory (os error 2)
[2024-04-23T06:58:11+08:00 ERROR util] crates/lsp/src/lsp.rs:768: oneshot canceled
[2024-04-23T06:58:11+08:00 ERROR project] failed to start language server "vue-language-server": Request initialize failed with message: Can't find typescript.js or tsserverlibrary.js in node_modules/typescript/lib
[2024-04-23T06:58:11+08:00 ERROR project] server stderr: Some("")
[2024-04-23T06:58:11+08:00 ERROR project] Hit 4 reinstallation attempts for "vue-language-server"
[2024-04-23T06:59:03+08:00 ERROR util] crates/lsp/src/lsp.rs:768: oneshot canceled
[2024-04-23T07:00:57+08:00 ERROR util] crates/terminal_view/src/terminal_view.rs:285: error upgrading view
[2024-04-23T07:01:00+08:00 ERROR util] crates/lsp/src/lsp.rs:768: oneshot canceled
```

### Environment

Zed: v0.132.2 (Zed Preview)
OS: macOS 14.4.1
Memory: 16 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

_No response_

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.

_No response_

--- a/extensions/vue/extension.toml
+++ b/extensions/vue/extension.toml
@@ -9,6 +9,7 @@ repository = "https://github.com/zed-industries/zed"
 [language_servers.vue-language-server]
 name = "Vue Language Server"
 language = "Vue.js"
+language_ids = { "Vue.js" = "vue" }
 # REFACTOR is explicitly disabled, as vue-lsp does not adhere to LSP protocol for code actions with these - it
 # sends back a CodeAction with neither `command` nor `edits` fields set, which is against the spec.
 code_action_kinds = ["", "quickfix", "refactor.rewrite"]

